helpers/getFontsCollection.js

```diff
@@ -3,8 +3,8 @@
 const getFonts = require('./lib/fonts');
 
 const factory = globals => {
-    return function() {
-        return getFonts('linkElements', globals.getThemeSettings(), globals.handlebars);
+    return function(options) {
```

Comment:
Do we need to document the parameter? Or do you guys usually just document it separately on DevDocs? Regardless, it might be a good practice to have a docblock showing how it could be applied (i.e.: `@example tag`). 🍺

Reply:
Changed it to the arguments pattern that I saw other people use.

Reply:
Right, I think it might be better if you keep the parameter name because it's easier to read. I just want a comment showing that it accepts 'font-display' as a valid optional parameter. At the moment, it's not immediately obvious unless you go through the code. 😄